apps/src/lib/kits/maker/MicroBitBoard.js

```diff
@@ -0,0 +1,105 @@
+/** @file Board controller for BBC micro:bit */
+/* global SerialPort */ // Maybe provided by the Code.org Browser
```

Comment:
Why maybe?

Reply:
If this code is being run in the browser, this value is provided by https://github.com/code-dot-org/browser/blob/a9bea833650028f4a34f211d02b12dbe042959b2/src/preload.js#L23